modules/core/site.js

```diff
@@ -2188,34 +2188,49 @@ function submitSmtpImapServer() {
         Hm_Notices.show(res.router_user_msgs);
 
         if (res.just_saved_credentials) {
-            $('#srv_setup_stepper_stepper').find('form').trigger('reset');
-            display_config_step(0);
-
-            //Initialize the form
-            $("#srv_setup_stepper_profile_reply_to").val('');
-            $("#srv_setup_stepper_profile_signature").val('');
-            $("#srv_setup_stepper_profile_name").val('');
-            $("#srv_setup_stepper_email").val('');
-            $("#srv_setup_stepper_password").val('');
-            $("#srv_setup_stepper_profile_is_default").prop('checked', true);
-            $("#srv_setup_stepper_is_sender").prop('checked', true);
-            $("#srv_setup_stepper_is_receiver").prop('checked', true);
-            $("#srv_setup_stepper_enable_sieve").prop('checked', false);
-            $("#srv_setup_stepper_only_jmap").prop('checked', false);
-            $('#step_config-imap_bloc').show();
-            $('#step_config-smtp_bloc').show();
-            $('#srv_setup_stepper_profile_bloc').show();
-
-            Hm_Utils.set_unsaved_changes(1);
-            Hm_Folders.reload_folders(true);
-            location.reload();
+            if (res.nux_server_id) {
```

Comment:
@Yannick243 just have some requests

Can you rename nux_server_id and nux_service_name as we are not in nux module here ?

Also what if imap_folders module is not enabled ? Can fix also for the nux module ?

Last thing we had a confirmation dialog before setting folders https://github.com/cypht-org/cypht/commit/ac3b9c283b8b8c5094f7362adacd8e0368c1c9d6#diff-3e95247a6dbbf437a9b5e7341db84da72c039b7b5ad532928b4cd8678dd5ff17R45. Shall we keep it for both or remove it for nux module too ? @marclaporte can you advise on this ? For my hand I'd prefer to remove it for nux module.

Reply:
I also prefer we remove this confirmation dialog. People can easily change the folders after if they wish (which is super rare).

Reply:
Hello @josaphatim,
The changes have been applied.